src/lava/lib/optimization/solvers/generic/builder.py

```diff
@@ -62,6 +62,7 @@ def verify_model_exists(self):
     def create_solver_process(
         self,
         problem: OptimizationProblem,
+        backend,

```

Comment:
For typing:
Is backend a 'AbstractComputeResource' or a 'str'?

Reply:
Als below

Reply:
Okay adding type hints in an upcoming commit